src/Riok.Mapperly/SyntaxProvider/Roslyn4_4_OrGreater.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Riok.Mapperly.Helpers;
+
+namespace Riok.Mapperly.SyntaxProvider;
+
+internal class Roslyn4_4_OrGreater
+{
+#if ROSLYN4_4_OR_GREATER
```

Comment:
Is there a reason why the precompiler directive here is only around the method but not around the entire file as in `Roslyn4_3_OrLower`?